frontend/src/components/customer/ProfileDetailsCard.tsx/ProfileDetailsCard.tsx

```diff
@@ -0,0 +1,480 @@
+                sx={{
+                    mt: "3rem",
+                    mb: "3rem",
+                }}
+            >
+                <Grid container direction="column">
+                    <Grid item sx={{ margin: "2rem" }}>
+                        <Heading
+                            variant="h2"
+                            color="secondary"
+                            style={{ marginLeft: "1rem" }}
+                        >
+                            Profile Details
+                        </Heading>
+                    </Grid>
+                    <Grid
+                        item
+                        sx={{
+                            ml: "2rem",
+                            mr: "2rem",
+                        }}
+                    >
+                        <Grid
+                            container
+                            direction="row"
+                            alignContent="flex-start"
+                            justifyContent="center"
+                            alignItems="center"
+                            spacing={3}
+                        >
+                            <Grid item xs={4}>
+                                <Grid container direction="column" gap={2}>
+                                    <Grid item>
+                                        <Avatar
+                                            src={profilePicture}
+                                            sx={{
+                                                width: "11rem",
+                                                height: "11rem",
+                                            }}
+                                        />
+                                    </Grid>
+                                    <Grid container direction="row" spacing={2}>
+                                        <Grid item>
+                                            <input
+                                                type="file"
+                                                accept="image/*"
+                                                id="profile-picture-upload"
+                                                style={{ display: "none" }}
+                                                onChange={handleFileUpload}
+                                            />
+                                            <label htmlFor="profile-picture-upload">
+                                                <Button
+                                                    variant="contained"
+                                                    component="span"
+                                                >
+                                                    Upload
+                                                </Button>
+                                            </label>
+                                        </Grid>
+                                        <Grid item>
+                                            {profilePicture && (
```

Comment:
Display button still even if no `profilePicture`, only passed it to be disabled will do